### DISCUSSION TITLE
Using Verdaccio fully offline

### DISCUSSION THREAD

**Discussion by Martisek** [verdaccio/verdaccio#2055] [Q&A] [answered]
Hello, 

I have got this question: is it possible and how to move Verdaccio from my development PC that has internet into an environment without connection?
 
I tried to make following steps however without positive result:
a) On second PC (Win10) I have installed Verdaccio and started it to run
b) On first PC (connected) I have created test React application (npx create-react-app test_app)
e) Copy test_app into second PC and publish it successfully 
f) Attempt to install published application - ERR! 404 Not found...

What is wrong? How make it properly?

Thank you a lot.

**Comment by juanpicado** [comment, marked as answer]
You might need to be more precise, I don't fully understand your setup.
